tests/configuration/verifyPMMSettingsPageFunctionality_test.js

```diff
@@ -1,4 +1,5 @@
 const { pmmSettingsPage } = inject();
+const { homePage } = inject();
```

Comment:
Why is there inject for homePage? That dependency is right now in the test on page 356